climpred/classes.py

```diff
@@ -604,27 +602,21 @@ def get_control(self):
         return self._datasets['control']
 
     def verify(
-        self,
-        metric='pearson_r',
-        comparison='m2e',
-        dim=None,
-        reference=None,
-        **metric_kwargs,
+        self, comparison=None, dim=None, metric=None, reference=None, **metric_kwargs,
```

Comment:
I like having them all as `None` keywords. just args would be an ordering hassle. maybe add a quick check that shouldnt be `None`?

Reply:
I am a bit unsure what we should do with `dim=None` because this actually will pass though xskillscore without failing for many metrics. Maybe we just leave it as `dim=None` and ignore. Should just be clear in the docstring then